spec/rubocop/cop/style/numeric_literal_prefix_spec.rb

```diff
@@ -0,0 +1,123 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::NumericLiteralPrefix, :config do
```

Comment:
I'd like to see some checks on what messages get output when reporting offenses (`cop.messages`) and what gets highlighted (`cop.highlights`).